src/components/Picture.jsx

```diff
@@ -1,17 +1,24 @@
 import React from "react";
 import PropTypes from "prop-types";
-import picturefill from "picturefill";
 import glamorous from "glamorous";
+import canUseDom from 'can-use-dom'
```

Comment:
Given that `can-use-dom` is [literally a function](https://github.com/akiran/can-use-dom/blob/master/index.js), what's your opinion on adding a dependency vs just defining that function in the library?

I know that npm is much more secure now and the chances something goes wrong are small, but given that the function defined isn't really something that we expect to change in the future I was more on the side of defining the function in our code instead of importing another dependency.